RADAR_LEADIN_ROUTINES.s

```diff
@@ -36,7 +36,7 @@
 # RADAR SAMPLING LOOP.
 
 		COUNT*	$$/RLEAD
-RADSAMP		CCS	RSAMPDT		# TIMES NORMAL ONCE PER SECOND SAMLING
+RADSAMP		CCS	RSAMPDT		# TIMES NORMAL ONCE PER SECOND SAMPLING
```

Comment:
Correction is correct. `ONCE PER SECOND` should be `ONCE-PER-SECOND` - [Source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0490.jpg)